src/lib.rs

```diff
@@ -1,18 +1,53 @@
+//! `swiftnav-rs` is a library that implements GNSS utility functions to perform
```

Comment:
perhaps add the LGPL file header; should be every file, but adding it to this one is a good start

Reply:
I've added it to all the source files, there aren't that many in this repo. Is that what you were thinking?

Reply:
yup